package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "commander": "2.9.0",
     "shell-quote": "1.8.1",
-    "which": "1.2.4",
+    "which": "^4.0.0",
```

Comment:
Version 4.0.0 drops Node 14 support, and does not have other significant changes, per https://github.com/npm/node-which/blob/main/CHANGELOG.md#400-2023-08-29

If we are planning to release a major version then this lgtm. Otherwise we should use 3.0.1 instead.

Considering that we are indeed preparing for a major version in https://github.com/mozilla/node-fx-runner/issues/89, 4.0.0 here is acceptable.